packages/rxfs/package.json

```diff
@@ -29,7 +29,7 @@
     "access": "public"
   },
   "scripts": {
-    "build": "rm -rf ./build/* && babel ./src -d ./build --ignore *.test.js",
+    "build": "BABEL_ENV=build babel src --out-dir build --source-maps --extensions .js --config-file ../../babel.config.js --delete-dir-on-start --no-comments",
     "lint": "eslint -c ../../.eslintrc ./src",
     "test": "mocha --exit ./src/**/*.test.js ./src/*.test.js",
     "test:watch": "yarn test -- --watch",
```

Comment:
I was running into a problem where modules were building as ES2015 modules, which don't play nice with older versions of node.js.  So I re-configured all of the modules to build as CommonJS modules which run on all recent versions of node.js and browsers.